app/src/main/java/com/kickstarter/libs/utils/extensions/BackingExt.kt

```diff
@@ -0,0 +1,30 @@
+@file:JvmName("BackingExt")
+package com.kickstarter.libs.utils.extensions
+
+import com.kickstarter.libs.utils.RewardUtils.isNoReward
+import com.kickstarter.libs.utils.RewardUtils.isShippable
+import com.kickstarter.models.Backing
+import com.kickstarter.models.Project
+import com.kickstarter.models.Reward
+
+    fun Backing.isBacked(reward: Reward): Boolean {
+        val rewardId = this.rewardId() ?: return isNoReward(reward)
+        return rewardId == reward.id()
+    }
+
+    fun Backing.isErrored(): Boolean  = this.status() == Backing.STATUS_ERRORED
+
+    fun Backing.isShippable(): Boolean {
+        val reward = this.reward() ?: return false
+        return isShippable(reward)
+    }
+
+    fun Backing.backedReward(project: Project): Reward? {
```

Comment:
Cn we add a test for this method?

Reply:
Whoops! You're correct. Thank you for reminding me!